lib/bin/main/WeatherSensorService/Internal/MockWeatherSensorReader.kt

```diff
@@ -0,0 +1,69 @@
+ //
+ // SpinDance
+ // Weather Sensor Service
+ //
+ // Copyright © 2022 SpinDance. All rights reserved.
+ //
+ 
+import kotlinx.coroutines.flow.MutableSharedFlow
+import kotlinx.coroutines.flow.SharedFlow
+import kotlinx.coroutines.flow.asSharedFlow
+import java.util.Timer
+import kotlin.concurrent.*
+import kotlin.random.Random
+import java.time.LocalDateTime
+
+private class Constants {
+    companion object {
+        var READER_INTERVAL: UInt = 1U
+        val TEMPERATURE_RANGE: ClosedRange<Double> = -40.0..40.0
+        val HUMIDITY_RANGE: ClosedRange<Double> = 0.0..100.0
+        val PRESSURE_RANGE: ClosedRange<Double> = 95.0..105.0
+    }
+}
+
+class MockWeatherSensorReader: WeatherSensorReaderType {
+     
+    private var timer: Timer? = null
+
+    override var readerInterval : UInt = Constants.READER_INTERVAL
+        
+    private val flow: MutableSharedFlow<WeatherSensorReading> = MutableSharedFlow<WeatherSensorReading>(replay = 1)
+    override var sensorReaderFlow : SharedFlow<WeatherSensorReading> = flow.asSharedFlow()
```

Comment:
Must have missed these before - Shouldn't be a space before the colon. Please do a search for ` : ` and clean those up.